src/Transmogrification.h

```diff
@@ -31,6 +31,13 @@ enum TransmogSettings
     SETTING_RETROACTIVE_CHECK = 1
 };
 
+enum MixedWeaponSettings
+{
+    MIXED_WEAPONS_STRICT = 0,
+    MIXED_WEAPONS_MODERN = 1,
+    MIXED_WEAPONS_LOOSE  = 2
```

Comment:
the enumeration starts always from 0, so you can also remove the assignment IIRC
```suggestion
    MIXED_WEAPONS_STRICT,
    MIXED_WEAPONS_MODERN,
    MIXED_WEAPONS_LOOSE
```

Reply:
I think having the values here may be helpful since they will show which setting corresponds to which config value.